### PR TITLE
docs: Update smart contract addresses (SC-14962)

### PR DESCRIPTION
**Pull request by vbidin**

### CODE REVIEW

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 | Contract | Address | Commit Hash |
 | -------- | ------- | ----------- |
-| Globals (Proxy) | [`0x7f3C3636208A18c7941BF051807db56864061465`](https://basescan.org/address/0x7f3C3636208A18c7941BF051807db56864061465) | [`globals-v2 @ v2.0.0`](https://github.com/maple-labs/globals-v2/releases/tag/v2.0.0) |
+| Globals (Proxy) | [`0x5A64417823E8382a7e8957E4411873FE758E73a8`](https://basescan.org/address/0x5A64417823E8382a7e8957E4411873FE758E73a8) | [`globals-v2 @ v3.0.0`](https://github.com/maple-labs/globals-v2/releases/tag/v3.0.0) |
```
Comment (JGcarv) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:7]: This address is for the globals proxy, so we shouldn't change it.

File: technical-resources/protocol-overview/smart-contract-addresses.md [hunk 2]
```diff
@@ -57,27 +57,36 @@
...
 | -------- | ------- | ----------- |
-| PoolManager | [`0xb918c805B33B02895F6F06c2EfA31d178678c947`](https://etherscan.io/address/0xb918c805B33B02895F6F06c2EfA31d178678c947) | [`pool-v2 @ v2.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v2.0.0) |
-| PoolManagerInitializer | [`0x0B240bf499773905802eE4DE43f96407C436d549`](https://etherscan.io/address/0x0B240bf499773905802eE4DE43f96407C436d549) | [`pool-v2 @ v2.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v2.0.0) |
+| PoolManager | [`0x0055c00ba4Dec5ed545A5419C4d430daDa8cb1CE`](https://etherscan.io/address/0x0055c00ba4Dec5ed545A5419C4d430daDa8cb1CE) | [`pool-v2 @ v3.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v3.0.0) |
+| PoolManagerInitializer | [`0x252C44A1630095504E3D3972b9b296f5ED494911`](https://etherscan.io/address/0x252C44A1630095504E3D3972b9b296f5ED494911) | [`pool-v2 @ v3.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v3.0.0) |
+| PoolManagerMigrator | [`0x9450d0D19802Ae0aDD44565752CDAB70E0A1C6ed`](https://etherscan.io/address/0x9450d0D19802Ae0aDD44565752CDAB70E0A1C6ed) | [`pool-v2 @ v3.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v3.0.0) |
```
Comment (JGcarv) [inline technical-resources/protocol-overview/smart-contract-addresses.md:62]: Should we add both migrators?

**Review by JGcarv — CHANGES_REQUESTED** [review]

File: technical-resources/protocol-overview/smart-contract-addresses.md [hunk 3]
```diff
@@ -154,7 +163,6 @@
 | WithdrawalManager (Proxy) | [`0x7f0d63e2250bc99f48985b183af0c9a66bbc8ac3`](https://etherscan.io/address/0x7f0d63e2250bc99f48985b183af0c9a66bbc8ac3) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
 | PoolDelegateCover (Proxy) | [`0x4Ab7F6F8DE3FE2575806a21F9Cac630DD51a84Af`](https://etherscan.io/address/0x4Ab7F6F8DE3FE2575806a21F9Cac630DD51a84Af) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
 
-
 ### Maven11 WETH Pool Contracts
```
Comment (0xfarhaan) [inline technical-resources/protocol-overview/smart-contract-addresses.md:166]: You are still missing the new opportunistic pool, please double check you have added all the correct pools

**Review by 0xfarhaan — CHANGES_REQUESTED** [review]
comments from me and JG

File: technical-resources/protocol-overview/smart-contract-addresses.md (hunk 2 above)
Comment (0xfarhaan) [reply technical-resources/protocol-overview/smart-contract-addresses.md:62]: yes lets add @vbidin

File: technical-resources/protocol-overview/smart-contract-addresses.md (hunk 3 above)
Comment (vbidin) [reply technical-resources/protocol-overview/smart-contract-addresses.md:167]: Added the opportunistic pool, noticed we're missing the old `laser` and `cicada` pools, are they relevant anymore?
Most of the pools on this list are deprecated or inactive.

Comment (0xfarhaan) [reply technical-resources/protocol-overview/smart-contract-addresses.md:167]: Lets add them so the list is exhaustive, worth having incase the frontend stops supporting a deprecated pool and someone needs the address to hit etherscan directly.

**Review by 0xfarhaan — APPROVED** [review]

**Pull request merged**